### PR TITLE
FLCN-158: UI tweaks. Loading spinner. Toasts.

### PR DESCRIPTION
**Pull request by danieltruong**

### CODE REVIEW

**Review by marklise — APPROVED** [review]

File: src/app/home/home.component.html [hunk 1]
```diff
@@ -86,7 +86,7 @@ <h1>Reserve a Day-Pass</h1>
         <div class="col"></div>
         <div class="col-4">
             <h2 class="py-2">
-                Select your Park
+                Select a Park
```
Comment (MaxWardle) [inline src/app/home/home.component.html:89]: noticed this banner should probably have the rounded edges too

File: src/assets/base/base.scss [hunk 2]
```diff
@@ -124,17 +137,17 @@ $theme-colors: (
 }
 
 .registration-details-container {
-    background-color: $bg-secondary;
-    border-radius: 5px;
+    // background-color: $bg-secondary;
```
Comment (MaxWardle) [inline src/assets/base/base.scss:140]: should get rid of these if not used

**Review by MaxWardle — APPROVED** [review]
looks good on mobile, approved with two small changes

File: src/assets/base/base.scss (hunk 2 above)
Comment (danieltruong) [reply src/assets/base/base.scss]: bdc7339

File: src/app/home/home.component.html (hunk 1 above)
Comment (danieltruong) [reply src/app/home/home.component.html:89]: bdc7339

**Pull request merged**